services/src/main/java/org/keycloak/authentication/authenticators/browser/ScriptBasedAuthenticator.java

```diff
@@ -49,7 +49,7 @@
  * <li>{@code user} the current {@link UserModel}</li>
  * <li>{@code session} the active {@link KeycloakSession}</li>
  * <li>{@code authenticationSession} the current {@link org.keycloak.sessions.AuthenticationSessionModel}</li>
- * <li>{@code httpRequest} the current {@link org.jboss.resteasy.spi.HttpRequest}</li>
+ * <li>{@code httpRequest} the current {@link org.keycloak.http.HttpRequest}</li>
```

Comment:
This got me thinking. The script authenticators are actually [supported](https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6/html/server_developer_guide/providers#authenticator) AFAICT. We'll need to also update the docs here and very, very explicitly mention this in migration guide (both prod and community).

This is not a breaking change to a private  SPI, this is a breaking change to public API. :/